mne/utils/docs.py

```diff
@@ -744,6 +744,14 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
     ``pos_lims``, as the surface plot must show the magnitude.
 """
 
+docdict[
+    "cmap"
+] = """
+cmap : matplotlib colormap | str | None
+        The :class:`~matplotlib.colors.Colormap` to use. Defaults to ``None``, meaning
+        the colormap will default to matplotlib's default.
```

Comment:
```suggestion
        The :class:`~matplotlib.colors.Colormap` to use. Defaults to ``None``, which
        will use the matplotlib default colormap.
```
technically this is still inaccurate, as I think it will use whatever is defined in the matplotlib RC file (which can be customized by the user).  But that's a subtlety that we don't need to get into here.